client/src/components/HeaderSearchField/HeaderSearchField.js

```diff
@@ -51,13 +54,39 @@ const StyledInputBase = styled(InputBase)(({ theme }) => ({
   },
 }));
 
-const HeaderSearchField = () => (
-  <Search>
-    <SearchIconWrapper>
-      <SearchIcon sx={{ color: appliedTheme.palette.text.primary }} />
-    </SearchIconWrapper>
-    <StyledInputBase placeholder="Search…" inputProps={{ 'aria-label': 'search' }} />
-  </Search>
-);
+const HeaderSearchField = () => {
+
+const [isOpenFilterItems, setIsOpenFilterItems] = useState(false)
+const handleFilter = () => {
+  setIsOpenFilterItems(false)
+}
+
+useEffect(() => {
+  if (isOpenFilterItems) {
+    setIsOpenFilterItems(true);
+   return  document.addEventListener('click', handleFilter);
+  }
```

Comment:
```suggestion
   document.addEventListener('click', handleFilter);
```

Reply:
done

Reply:
@valeriaholdovych  nope it is not done)   no need for the if condition.. it doesnt make sense. please go through it again